### PR TITLE
4.1.1 - metadata - Test 5

### PR DESCRIPTION
**Pull request by D-Cobbledick**

### CODE REVIEW

File: _indicators/4-1-1.md [hunk 1]
```diff
@@ -3,83 +3,135 @@ indicator: 4.1.1
...
-  outcomes
-reporting_status: complete
-published: true
-source_active_1: true
-source_url_text_1: Key Stage 1 data
-source_active_2: true
-source_url_text_2: Key Stage 2 data
-source_active_3: true
-data_non_statistical: false
+  4.1 By 2030, ensure that all girls and boys complete free, equitable and quality primary and secondary education leading to relevant and effective learning outcomes
```
Comment (dougmet) [inline _indicators/4-1-1.md]: Need to make sure the target ID is not being put in the title

File: _indicators/4-1-1.md [hunk 2]
```diff
@@ -3,83 +3,135 @@ indicator: 4.1.1
 layout: indicator
 permalink: /4-1-1/
 sdg_goal: '4'
-title: "Proportion of children and young people: (a)\_in\_grades 2/3; (b) at the end of primary; and (c)\_at\_the end of lower secondary achieving at least a minimum proficiency level in (i) reading and (ii)\_mathematics, by sex"
-graph_type: line
-un_custodian_agency: >-
-  United Nations Educational, Scientific and Cultural Organization - Institute
-  for Statistics (UNESCO-UIS)
+title: >-
+  4.1.1 Proportion of children and young people: (a) in grades 2/3; (b) at the end of primary; and (c) at the end of lower secondary achieving at least a minimum proficiency level in (i) reading and (ii) mathematics, by sex
```
Comment (dougmet) [inline _indicators/4-1-1.md]: Need to make sure hat the indicator ID is not in the title

File: _indicators/4-1-1.md [hunk 3]
```diff
@@ -3,83 +3,135 @@ indicator: 4.1.1
...
-published: true
-source_active_1: true
-source_url_text_1: Key Stage 1 data
-source_active_2: true
-source_url_text_2: Key Stage 2 data
-source_active_3: true
-data_non_statistical: false
+  4.1 By 2030, ensure that all girls and boys complete free, equitable and quality primary and secondary education leading to relevant and effective learning outcomes
+indicator_name: >-
+  4.1.1 Proportion of children and young people: (a) in grades 2/3; (b) at the end of primary; and (c) at the end of lower secondary achieving at least a minimum proficiency level in (i) reading and (ii) mathematics, by sex
```
Comment (dougmet) [inline _indicators/4-1-1.md]: Remove indicator ID from the indicator name

Comment (D-Cobbledick) [reply _indicators/4-1-1.md]: Thanks Doug, I hope to look at this again later today

**Pull request closed without merging**